src/LuaAST.ts

```diff
@@ -41,58 +41,73 @@ export enum SyntaxKind {
     TableIndexExpression,
     // Operators
     // Arithmetic
-    AdditionOperator,  // Maybe use abbreviations for those add, sub, mul ...
+    AdditionOperator, // Maybe use abbreviations for those add, sub, mul ...
     SubtractionOperator,
     MultiplicationOperator,
     DivisionOperator,
     FloorDivisionOperator,
     ModuloOperator,
     PowerOperator,
-    NegationOperator,  // Unary minus
+    NegationOperator, // Unary minus
     // Concat
     ConcatOperator,
     // Length
-    LengthOperator,  // Unary
+    LengthOperator, // Unary
     // Relational Ops
     EqualityOperator,
     InequalityOperator,
     LessThanOperator,
     LessEqualOperator,
-    GreaterThanOperator,   // Syntax Sugar `x > y` <=> `not (y <= x)`
-                           // but we should probably use them to make the output code more readable
-    GreaterEqualOperator,  // Syntax Sugar `x >= y` <=> `not (y < x)`
+    GreaterThanOperator, // Syntax Sugar `x > y` <=> `not (y <= x)`
+    // but we should probably use them to make the output code more readable
```

Comment:
We should probably avoid aligning comment like this and put them on their own lines.